Content.Shared/Materials/MaterialPrototype.cs

```diff
@@ -42,5 +42,11 @@ public sealed class MaterialPrototype : IPrototype, IInheritingPrototype
         [ViewVariables]
         [DataField("icon")]
         public SpriteSpecifier Icon { get; } = SpriteSpecifier.Invalid;
+
+        /// <summary>
+        /// The price per cm3.
+        /// </summary>
+        [DataField("price", required: true)]
+        public double Price = 0;
```

Comment:
Adding an antag specific field to materialprototype is kinda ass.

Reply:
If it's for cargo shuttle it's probably okay but in that case some of the other code would need tweaking.

Reply:
This is for cargo too.